Python/Tests/TestAdapterTests/Mocks/MockRunSettingsXmlBuilder.cs

```diff
@@ -54,18 +55,20 @@ class MockRunSettingsXmlBuilder {
         private string _resultsDir;
         private string _testDir;
         private string _interpreterPath;
+        private string _discoveryWaitTimeInSeconds;
         private StringBuilder _environmentLines = new StringBuilder();
         private StringBuilder _searchLines = new StringBuilder();
         private StringBuilder _testLines = new StringBuilder();
 
-        public MockRunSettingsXmlBuilder(string testFramework, string interpreterPath, string resultsDir, string testDir) {
+        public MockRunSettingsXmlBuilder(string testFramework, string interpreterPath, string resultsDir, string testDir, string discoveryWaitTimeInSeconds = default ) {
```

Comment:
This would be better as an integer. You could make it default to -1 and send string.empty if < 0 when you format the string.

Reply:
will do